releasenotes/notes/[Docker-check]-fixes:-cgroup-parsing-+-docker.mem.in_use-computation-e390c8f2c2541d2f.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
```

Comment:
Could you please make two items, like this:
```
fixes:
  - Fix docker memory limit metrics parsing from cgroup files
  - Fix docker.mem.in_use metric computation
```

Reply:
👍